client/src/pages/LogInForm.js

```diff
@@ -27,6 +27,7 @@ function LogInForm() {
 
 			localStorage.setItem("authToken", token);
 			localStorage.setItem("userType", user_type);
+			setRole(user_type);
```

Comment:
This is not the right way to do this. Returning users have a role without having to log in again.

Revert changes to this file. They are not needed.